modules/backup-etcd.adoc

```diff
@@ -22,6 +22,14 @@ Only save a backup from a single master host. Do not take a backup from each mas
 ====
 You can check whether the proxy is enabled by reviewing the output of `oc get proxy cluster -o yaml`. The proxy is enabled if the `httpProxy`, `httpsProxy`, and `noProxy` fields have values set.
 ====
+* If the scheduler has a default node selector set, create a temporary project that overwrites the default node selector:
```

Comment:
probably something like this instead, now that i realize most installer-created namespaces already have the necessary annotation:
```
If the scheduler has a default node selector set, to debug a node that does not have labels matching
the default node selector, you must run the debug command from a namespace that has this annotation:
    openshift.io/node-selector: ""
The empty node-selector annotation is already set in many installer-created namespaces.  For example, 
`openshift-apiserver`, `openshift-config`,  and `openshift-console` have the empty 
openshift.io/node-selector annotation that allows pods to be created regardless of a 
defaultNodeSelector that may be configured in the scheduler.  
Or, you can create a temporary project with that annotation with the following: 
$ oc adm new-project <temporary-project> --node-selector="" from which you will run the `oc debug node` command.
``` 
```

Reply:
I like ^ wording. I'd also add a link to https://docs.openshift.com/container-platform/4.5/nodes/scheduling/nodes-scheduler-default.html so that people can look at that for reference.

Reply:
-1 on docs.  We should just make it work.